projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -4136,6 +4136,51 @@ def ElementwiseBitwiseAndScalarInt8Module_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ElementwiseQuantizePerTensorModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float, True),
+    ])
+    def forward(self, x):
+        scale = 0.04
+        zp = -110
+        dtype = torch.qint8
+        q = torch.quantize_per_tensor(x, scale, zp, dtype)
+        return q.int_repr()
+
+@register_test_case(module_factory=lambda: ElementwiseQuantizePerTensorModule())
+def ElementwiseQuantizePerTensorModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4))
+
+
+# ==============================================================================
+
+class ElementwiseDequantizePerTensorModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.int8, True),
+    ])
+    def forward(self, x):
+        qx = torch._make_per_tensor_quantized_tensor(x, 0.1, 8)
+        qx = torch.dequantize(qx)
+        return qx
+
+@register_test_case(module_factory=lambda: ElementwiseDequantizePerTensorModule())
+def ElementwiseDequantizePerTensorModule_basic(module, tu: TestUtils):
+    module.forward(tu.randint(3, 4, low=-128, high=127).to(torch.int8))
+
+# ==============================================================================
```

Comment:
Out of curiosity, what is `aten.dequantize_self_per_tensor`? Is it worth trying to test here?

Reply:
Its the same as `aten.quantize_per_tensor`. The only difference is whether its the lib call or member call. AFAICT we cannot actually generate the none-self version.